config/environments/staging.rb

```diff
@@ -3,12 +3,12 @@
 Rails.application.configure do
   config.active_storage.service = :s3_staging
 
-  config.ctc_url = "https://ctc.staging.getyourrefund.org"
-  config.statefile_url = "https://staging.fileyourstatetaxes.org"
-  config.gyr_url = "https://staging.getyourrefund.org"
-  ctc_email_from_domain = "getctc.org"
-  gyr_email_from_domain = "getyourrefund.org"
-  statefile_email_from_domain = "fileyourstatetaxes.org"
+  config.ctc_url = "https://ctc.demo.getyourrefund.org"
```

Comment:
Did we get rid of staging already? I might just be out of the loop, but what is the basis for these changes?